common/workunit/workunit.cpp

```diff
@@ -13883,6 +13888,18 @@ extern WORKUNIT_API void addTimeStamp(IWorkUnit * wu, StatisticScopeType scopeTy
     wu->setStatistic(queryStatisticsComponentType(), queryStatisticsComponentName(), scopeType, scopestr, kind, NULL, getTimeStampNowValue(), 1, 0, StatsMergeAppend);
 }
 
+extern WORKUNIT_API void addTimeStamp(IWorkUnit * wu, unsigned wfid, const char * scope, StatisticKind kind)
+{
+    StringBuffer scopestr;
```

Comment:
`addTimeStamp(wu, getScopeType(scope), scope, kind, wfid);
`
?

Reply:
that would mean it performs getScopeType on the passed in 'scope', before it had prepended it with the (e.g.) "w1"

Reply:
Yes, that's ok - because that function only looks at the trailing element in the scope to determine the type.  e.g. w1:graph1:sg3 is a subgraph

Reply:
ah ok, yes I see that now.
I'll change it.